docs/en_US/guide/ai/python_openvino_tutorial.md

```diff
@@ -0,0 +1,286 @@
+
+    ```python
+    if __name__ == '__main__':
+        # Define the plugin
+        c = PluginConfig("defect", {}, {},
+                         {"inference": lambda: inferenceIns})
+        # Start the plugin instance
+        plugin.start(c)
+    ```
+
+3. Create a plugin description file in JSON format to define the metadata of the plugin. The file name must be the same as the plugin name, i.e. `defect.json`. The function names defined therein must correspond exactly to the entry file, and the contents of the file are as follows. Where, executable is used to define the name of the plugin's executable entry file.
+
+    ```json
+    {
+      "version": "v1.0.0",
+      "language": "python",
+      "executable": "main.py",
+      "sources": [
+      ],
+      "sinks": [
+      ],
+      "functions": [
+        "inference"
+      ]
+    }
+    ```
+
+At this point we have completed the development of the plugin, next we just need to package all the files in the directory into a zip file. The zip file should have a file structure similar to:
+
+- inference.py
+- requirements.txt
+- install.sh
+- main.py
+- defect.json
+- models
+  - model.bin
+  - model.xml
+- functions
+  - defect.json
+
+## Plugin Installation
+
+As the same with installing native plugins, we can also install Python plugins via the eKuiper manager UI or the REST API. To use the REST API, upload the zip file packaged above to the machine where eKuiper is located. Then use the following API to install it:
+
+```text
+### Install pyai plugin
+POST http://{{host}}/plugins/functions
+Content-Type: application/json
+
+{"name":"defect", "file": "file:///tmp/defect.zip"}
+```
+
+The installation process requires an Internet connection to download dependencies, including `ffmpeg libsm6 libxext6`, which may take a long time depending on the network conditions.
+
+## Run the plugin
+
+Once the plugin installed, we can use it in our rule. We will create a rule to receive image byte data from a mqtt topic and do the inference for the image by OpenVINO model.
+
+### Define the stream
+
+Define the stream by eKuiper rest API. We create a stream named openvino_demo and the topic is openvino_demo.
+
+```shell
+POST http://{{host}}/streams
+Content-Type: application/json
+
+{"sql":"CREATE STREAM openvino_demo () WITH (DATASOURCE=\"openvino_demo\")"}
+```
+
+### Define the rule
+
+Define the rule by eKuiper rest API.  We will create a rule named ruleOp. We just read the base64 encoded images from openvino_demo stream and run the custom function *inference* against it.  
+It will send out the base64 encoded origin and processed image to topic *ekuiper/defect* when the segment defects numbers not be zero.
```

Comment:
```suggestion
If the number of segmented defects is non-zero, it will dispatch the base64 encoded original and processed images to the `ekuiper/defect` topic.
```

Reply:
I think these 2 sentences belong to one paragraph.